airflow/models/dag.py

```diff
@@ -1384,7 +1390,8 @@ def sync_to_db(self, owner=None, sync_time=None, session=None):
         orm_dag.default_view = self._default_view
         orm_dag.description = self.description
         orm_dag.schedule_interval = self.schedule_interval
-        session.merge(orm_dag)
+        orm_dag.tags = [self.get_dagtag(name, session=session) for name in set(self.tags)]
```

Comment:
To avoid multiple calls to the DB, could we add a convenience method `get_dagtags` for supplying a list of tags? Or change `get_dagtag` to accept a list of strings besides a single string?